webportal/src/app/market_list/components/side_bar.jsx

```diff
@@ -1,62 +1,88 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-import React, { useContext } from 'react';
+import React, { useContext, useState } from 'react';
 import { getTheme, Stack, Text, SearchBox } from 'office-ui-fabric-react';
-import PropTypes from 'prop-types';
-import FilterItem from './filter_item';
+import queryString from 'query-string';
+import { isNil } from 'lodash';
+import TypeFilter from './type_filter';
 import Context from 'App/context';
 import { TYPE_ENUM } from 'App/utils/constants';
 
-const SideBar = props => {
-  const { type } = props;
+const SideBar = () => {
+  const [type, setType] = useState(null);
```

Comment:
Is here change type propagation from props to redux ?

Reply:
change props to state by using react hook